lib/utils.js

```diff
@@ -23,25 +24,21 @@ exports.walk = function(dir) {
 };
 
 // Walk a directory and remove every folder that is empty inside it
-function removeEmptyDirectories(fs, directory, callback) {
-  fs.stat(directory, function(err, stat) {
-    if (err) return callback(err);
-    if (!stat.isDirectory()) return callback();
-    fs.readdir(directory, function(err, list) {
-      if (err) return callback(err);
-      if (list.length === 0) {
-        return fs.rmdir(directory, callback);
-      }
+exports.removeEmptyDirectories = function(fs, directory, done) {
```

Comment:
Arrow function?

Reply:
This is the only place where I do have opinion on this. It's a top-level function, so the `this` context will never be relevant. With ESM modules, I also just prefer how
```javascript
export function removeEmptyDirectories(fs, directory, done) { ... }
```
looks compared to
```javascript
export const removeEmptyDirectories = (fs, directory, done) => { ... }
```
The only place I actually prefer arrow functions is when they are passed directly into callbacks, since those are the only places I've found that `this` is actually relevant (with a few exceptions).

If this seems too inconsistent I can update it everywhere, there doesn't need to be much debate.